### PR TITLE
[15.0][MIG] contract_variable_quantity: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT40289

### CODE REVIEW

File: contract_variable_quantity/tests/test_contract_variable_quantity.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
...
 
 from odoo import exceptions
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
-class TestContractVariableQuantity(SavepointCase):
+class TestContractVariableQuantity(TransactionCase):
     at_install = False
     post_install = True
```
Comment (CarlosRoca13) [inline contract_variable_quantity/tests/test_contract_variable_quantity.py]: I think it should be used with the decorator `@odoo.tests.tagged("post_install", "-at_install")`

**Review by pedrobaeza — APPROVED** [review]
Tested on runboat.

**Review by CarlosRoca13 — APPROVED** [review]
Thanks

**Pull request merged**